packages/react-composites/src/composites/common/Drawer/SpokenLanguageDrawer.styles.ts

```diff
@@ -10,12 +10,23 @@ import { _DrawerMenuStyles } from '@internal/react-components';
  */
 export const spokenLanguageDrawerStyles = (theme: Theme): _DrawerMenuStyles => ({
   root: {
-    height: _pxToRem(300),
+    height: '100%',
     overflow: 'auto'
   },
   drawerSurfaceStyles: {
+    root: {
+      // Targets FocusTrapZone in drawer specific to spokenLanguageDrawer so it does not impact
+      // components using FocusTrapZone.
+      // Setting percentage to Height to transform a container does not work unless the
+      // direct parent container also has a Height set other than 'auto'.
+      '> div:nth-child(2)': {
+        height: '75%',
```

Comment:
I wonder if we are to apply this to all drawers that this should be max height and let the other drawers shrink if need be?

Reply:
That's right, after I surfaced it to the DrawerSurface Component I switched it to maxHeight.